app/scripts/views/sign_in.js

```diff
@@ -95,6 +96,12 @@ define(function (require, exports, module) {
       this._formPrefill.set('password', this.getElementValue('.password'));
     },
 
+    hideSuccessAndErrorMessages: function () {
+      $(this.el).off('keypress', '.input-row');
```

Comment:
I unbound the listener so it won't catch the event for every key stroke. If I don't do this I'll have a function running for every keypress in the input. So my line of thought was that hiding it once the user starts typing was enough. So,
1) would you like me to leave it bound for every key stroke?
2) Sorry, but I didn't understand why it's better to move the event to FormView. I thought this was a change to be made in a smaller scope (sign in view) and not for all the other view inheriting from FormView. I can see this is going to work, but I didn't get why this is better practice.
Thanks again for your attention.